src/gdcdictionary/schemas/other_clinical_attribute.yaml

```diff
@@ -0,0 +1,1259 @@
+$schema: "http://json-schema.org/draft-04/schema#"
+
+id: "other_clinical_attribute"
+title: Other Clinical Attribute
+type: object
+namespace: https://gdc.cancer.gov
+category: clinical
+project: '*'
+program: '*'
+description: >-
+  Additional clinical attributes that are extraneous to other specific clinical entity
+  types within the GDC Data Dictionary.
+additionalProperties: false
+submittable: true
+downloadable: false
+previous_version_downloadable: false
+validators: null
+
+systemProperties:
+  - id
+  - project_id
+  - created_datetime
+  - updated_datetime
+  - state
+
+links:
+  - exclusive: false
+    required: true
+    subgroup:
+      - name: cases
+        backref: other_clinical_attributes
+        label: describes
+        target_type: case
+        multiplicity: many_to_one
+        required: false
+      - name: follow_ups
+        backref: other_clinical_attributes
+        label: describes
+        target_type: follow_up
+        multiplicity: many_to_one
+        required: false
+
+required:
+  - submitter_id
+
+uniqueKeys:
+  - [id]
+  - [project_id, submitter_id]
+
+properties:
+
+  $ref:
+    - "_definitions.yaml#/ubiquitous_properties"
+
+  aids_risk_factors:
+    $ref:
+      - "_terms.yaml#/aids_risk_factors/common"
+    enum:
+      - Candidiasis
+      - Coccidioidomycosis
+      - Cryptococcosis
+      - Cryptosporidiosis, Chronic Intestinal
+      - Cytomegalovirus
+      - Encephalopathy
+      - Herpes Simplex Virus
```

Comment:
Yeah the actual binomial is Human alphaherpesvirus 1 or Human alphaherpesvirus 2 it seems like, I think we can leave it as is since it is not the true binomial, more like the common name I guess.